src/main/java/com/daitangroup/api/services/ProductionOrderService.java

```diff
@@ -0,0 +1,105 @@
+package com.daitangroup.api.services;
+
+import com.daitangroup.api.model.Person;
+import com.daitangroup.api.model.ProductionOrder;
+import com.daitangroup.api.model.Vehicle;
+import com.daitangroup.api.repository.PersonRepository;
+import com.daitangroup.api.repository.ProductionOrderRepository;
+import com.daitangroup.api.repository.VehicleRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+
+@Service
+public class ProductionOrderService {
+
+    @Autowired
+    private PersonRepository personRepository;
+
+    @Autowired
+    private ProductionOrderRepository productionOrderRepository;
+
+    @Autowired
+    private VehicleRepository vehicleRepository;
+
+    public Optional<ProductionOrder> findById(long id) {
+        return productionOrderRepository.findById(id);
+    }
+
+    public Iterable<ProductionOrder> findAll() {
+        return productionOrderRepository.findAll();
+    }
+
+    public ProductionOrder createProductionOrder(long personid, long vehicleid, ProductionOrder productionOrder) {
+
+        Optional<Person> person = personRepository.findById(personid);
+        Optional<Vehicle> vehicle = vehicleRepository.findById(vehicleid);
+
+        if(person.isPresent() && vehicle.isPresent()) {
+            productionOrder.setVehicle(vehicle.get());
+            productionOrder.setPerson(person.get());
+
+            return productionOrderRepository.save(productionOrder);
+        }
+
+        return null;
+    }
+
+    public ProductionOrder changeProductionOrder(long productionId, Optional<Long> personid, Optional<Long> vehicleid, ProductionOrder productionOrder) {
```

Comment:
Rename it to updateProductionOrder

Can't we use the production order object to carry the IDs?